nepal_compliance/nepal_compliance/report/sales_return_register_ird/sales_return_register_ird.py

```
@@ -0,0 +1,114 @@
+            si.posting_date
+        FROM `tabSales Invoice` si
+        WHERE {' AND '.join(conditions)}
+        ORDER BY si.posting_date
+    """
+
+    invoices = frappe.db.sql(query, values, as_dict=True)
+    data = []
+
+    for inv in invoices:
+        customer_country = frappe.db.get_value("Customer", inv.customer_name, "territory") or ""
+        is_export = customer_country.strip().lower() not in ("", "nepal")
```

Ruff (0.11.9)

Local variable `is_export` is assigned to but never used

Remove assignment to unused variable `is_export`

(F841)